tests/testSkyModel.py

```diff
@@ -160,7 +158,7 @@ def test90Deg(self):
         sm = self.sm_mags
         sm.setRaDecMjd(0., 90., mjd, degrees=True, azAlt=True)
         mags = sm.returnMags()
-        for key in mags.dtype.names:
+        for key in mags.keys():
```

Comment:
I think to be easier to translate to python3, you can just do 'for key in mags:'  (the iterator over mags is over the keys).